server/src/services/lbajob.service.ts

```diff
@@ -0,0 +1,265 @@
+  } catch (error) {
+    return manageApiError({ error, api_path: "jobV1/matcha", caller, errorTitle: "getting job by id from Matcha" })
+  }
+}
+
+/**
+ * Adaptation au modèle LBAC et conservation des seules infos utilisées de l'offre
+ * @param {Partial<IRecruiter>} job l'offre retournée d'ES ou de la mongo
+ * @param {number} distance optionnel: la distance du lieu de l'offre au centre de la recherche
+ * @param {string} caller optionnel: l'id de l'utilisateur de l'api
+ * @param {IApplicationCount[]} applicationCountByJob le tableau des décomptes de candidatures par offre
+ * @returns {ILbaItem[]}
+ */
+const transformLbaJob = ({
+  job,
+  distance,
+  caller,
+  applicationCountByJob,
+}: {
+  job: Partial<IRecruiter>
+  distance?: number
+  caller?: string
+  applicationCountByJob: IApplicationCount[]
+}): ILbaItem[] => {
+  const resultJobs = []
+
+  job.jobs.map((offre, idx) => {
+    const resultJob = new LbaItem("matcha")
+    let email = {}
+
+    email = encryptMailWithIV({ value: job.email, caller })
+
+    resultJob.id = `${job.establishment_id}-${idx}`
+    resultJob.title = offre.rome_appellation_label ?? offre.rome_label
+    resultJob.contact = {
+      ...email,
+      name: job.first_name + " " + job.last_name,
+      phone: job.phone,
+    }
+
+    resultJob.place.distance = distance ? roundDistance(distance) : null
+    resultJob.place.fullAddress = job.address
+    resultJob.place.address = job.address
+    resultJob.place.latitude = job.geo_coordinates.split(",")[0]
+    resultJob.place.longitude = job.geo_coordinates.split(",")[1]
+
+    resultJob.company.siret = job.establishment_siret
+    resultJob.company.name = job.establishment_enseigne || job.establishment_raison_sociale || "Enseigne inconnue"
+    resultJob.company.size = job.establishment_size
+
+    resultJob.company.mandataire = job.is_delegated
+
+    resultJob.company.place = { city: job.establishment_location }
+
+    resultJob.nafs = [{ label: job.naf_label }]
+    resultJob.company.creationDate = job.establishment_creation_date
+
+    resultJob.diplomaLevel = offre.job_level_label
+    resultJob.createdAt = job.createdAt
+    resultJob.lastUpdateAt = job.updatedAt
+
+    resultJob.job = {
+      id: offre._id,
+      description: offre.job_description || "",
+      creationDate: offre.job_creation_date,
+      contractType: offre.job_type.join(", "),
+      jobStartDate: offre.job_start_date,
+      romeDetails: offre.rome_detail,
+      rythmeAlternance: offre.job_rythm || null,
+      dureeContrat: "" + offre.job_duration,
+      quantiteContrat: offre.job_count,
+      elligibleHandicap: offre.is_disabled_elligible,
+    }
+
+    resultJob.romes = []
+    offre.rome_code.map((code) => resultJob.romes.push({ code, label: null }))
+
+    const applicationCount = applicationCountByJob.find((job) => job._id == offre._id)
+    resultJob.applicationCount = applicationCount?.count || 0
+    resultJobs.push(resultJob)
+  })
+
+  return resultJobs
```

Comment:
```suggestion
     return resultJob
  })
```
va avec la suggestion plus haut